demos/demo_simulate_a_robot.py

```diff
@@ -20,7 +20,11 @@
 
 from bullet_utils.env import BulletEnvWithGround
 from robot_properties_solo.solo8wrapper import Solo8Robot
-import importlib.resources
+try:
```

Comment:
This is unnecessary here as importlib is not being used in this file.

Reply:
Indeed you should remove the import altogether